os/mm/mm_heap/mm_check_heap_corruption.c

```diff
@@ -96,9 +96,9 @@ static void dump_node(struct mm_allocnode_s *node, node_type_t type)
 #endif
 
 	if (type == TYPE_CORRUPTED) {
-		dbg("CORRUPTED NODE: addr = 0x%08x size = %u preceding size = %u\n", node, node->size, MM_PREV_NODE_SIZE(node));
+		lldbg("CORRUPTED NODE: addr = 0x%08x size = %u preceding size = %u\n", node, node->size, MM_PREV_NODE_SIZE(node));
```

Comment:
Let's use mfdbg like others.

Reply:
I want to keep the same policy. Sometimes we use mfdbg, but others lldbg. What I suggest is that `mfdbg` is memory fault debug message. This function is for that. That's why I suggested. How do you think?